doc/sdk-cli-reference.md

```diff
@@ -143,6 +143,34 @@ pkg/apis/app/v1alpha1/
 └── zz_generated.deepcopy.go
 ```
 
+### openapi
+
+Runs the [kube-openapi][openapi-code-generator] OpenAPIv3 code generator for all Custom Resource Definition (CRD) API tagged fields under `pkg/apis/...`.
+
+**Note**: This command must be run every time the API (spec and status) for a custom resource type is updated.
```

Comment:
Validation only needs to be regenerated when the spec block changes right? Not the status block.